src/collections/_documentation/performance/performance-tab.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Performance Tab'
+sidebar_order: 1
+---
+
+[SCREENSHOT of Performance Homepage]
+
+The Performance Homepage offers Apdex and Throughput graphs as well as a list of the most problematic transactions so users can see a high-level report of their overall application health.
+
+## Apdex & Throughput Graphs
+The Apdex graph shows the ratio of response times in a user's application against a threshold. Users can get, on aggregate, a sense of how long a customer has waited. For example, if there is a significant dip in an Apdex score in a given window, a user may want to dig into that time period to investigate a potential performance bottleneck.
```

Comment:
The term "user" has two different meanings here. The first is the user of the application, the second is the user of Sentry. 

Suggest: "The Apdex graph displays the ratio of response times in an application measured against a threshold. This information is displayed as an aggregate so you can get a sense of how long a customer has waited. For example, if the Apdex score dips significantly during a given time period, you may want to investigate that time period for a potential performance bottleneck."